src/collapse/perf.rs

```diff
@@ -609,7 +609,14 @@ fn tidy_generic(mut func: String) -> String {
     //    see https://github.com/brendangregg/FlameGraph/pull/72
     //  - C++ anonymous namespace annotations.
     //    see https://github.com/brendangregg/FlameGraph/pull/93
-    if let Some(first_paren) = func.find('(') {
+
+    // Check for C++ template parameters
+    let mut offset = 0usize;
+    if let Some(last_closing_angle_bracket) = func.rfind('>') {
```

Comment:
I don't think this will quite work either. Consider, for example:
```rust
foo(Vec<usize>)
```

Reply:
True, probably actually need to do some recursive descent parsing here.